sweep/walletsweep.go

```diff
@@ -152,7 +152,7 @@ func (f FeeEstimateInfo) Estimate(estimator chainfee.Estimator,
 // UtxoSource is an interface that allows a caller to access a source of UTXOs
 // to use when crafting sweep transactions.
 type UtxoSource interface {
-	// ListUnspentWitness returns all UTXOs from the default wallet account
+	// ListUnspentWitnessFromDefaultAccount returns all UTXOs from the default wallet account
```

Comment:
wrap lines to 80 columns otherwise the linter will complain.

Reply:
Thanks for your guidance, I resized the length of each line.